app/views/devise/mailer/reset_password_instructions.html.erb

```diff
@@ -81,6 +81,19 @@
                                                 </table>
                                               </td>
                                             </tr>
+                                            <tr>
+                                              <td>
+                                                <table width="100%" border="0" cellspacing="0" cellpadding="0">
+                                                  <tr>
+                                                    <td align="left" style="font-family: Helvetica,Arial,sans-serif; box-sizing: border-box; font-size: 14px; vertical-align: top; margin: 0; padding: 0 0 20px;" class="padding">
+                                                      Your password must be reset within the next 6 hours. If you do not
+                                                      reset your password within the next 6 hours, request a new password
+                                                      reset message from the login page.
```

Comment:
Can you use a variable that references the config value? That way if the configuration value changes, the text of this email will be correct.  See [this SO post](https://stackoverflow.com/questions/11231687/accessing-devise-config-variables) for an example of how to access the Devise config variables programmatically.

Reply:
I completely overlooked this request when I commented last week, great idea!